### ISSUE TITLE
[Tech Debt] Convert Decorator Inputs/Queries to Signals / Move to all Standalone

### ISSUE THREAD

**Issue by johnsonav1992** [eliotmjohnson/BudgetBros#133]
We should move everything to the newest (and officially recommended by Angular team) conventions to support future versions and scalability.

- [ ] Viewchild, Viewchildren, Contentchild, Contentchildren to signal based APIs
- [ ] Input and Output to signal based as well as EventEmitters to output()s
- [ ] All standalone components/pipes and remove NgModules

**Comment by eliotmjohnson** [comment]
Can we split this into my work on the budgets side and yours on the trx side? I'm not ready to move towards this on the budgets side. Thanks.

**Comment by johnsonav1992** [comment]
Is there something specifically blocking you there?

**Comment by eliotmjohnson** [comment]
Nope, I’m just going to do all upgrades incrementally on the budgets side. You should still be able to do standalone components for the trx side while still keeping the main module.

**Comment by johnsonav1992** [comment]
I think I will wait to finish this issue then until budgets is stable because I feel like it would be easier to do in one fell swoop.

**Comment by eliotmjohnson** [comment]
Okay, in the meantime, I'm going to incrementally move budgets infrastructure over to signals as much as possible.
